started to fail testing in datalad-extensions

about 3 weeks ago, see e.g first one to fail
https://github.com/datalad/datalad-extensions/actions/runs/3814859295/jobs/6489457209
```
FAILED tests/test_workflow.py::test_workflow_new - AssertionError
```

--- a/datalad_catalog/extractors/datacite_gin.py
+++ b/datalad_catalog/extractors/datacite_gin.py
@@ -17,8 +17,8 @@ from datalad_metalad.extractors.base import (
     DataOutputCategory,
     ExtractorResult,
 )
+from datalad_metalad.extractors.legacy.definitions import vocabulary_id
 from datalad.log import log_progress
-from datalad.metadata.definitions import vocabulary_id
 from datalad.utils import assure_unicode
 
 

--- a/setup.cfg
+++ b/setup.cfg
@@ -15,7 +15,7 @@ classifiers =
 python_requires = >= 3.7
 install_requires =
     datalad >= 0.17
-    datalad-metalad >= 0.4.1
+    datalad-metalad >= 0.4.6
     jsonschema
     pyyaml
     jq

